cli/reference/blocks/globals.md

```diff
@@ -4,3 +4,48 @@ description: Learn how to define Globals using the globals block.
 ---
 
 # The `globals` block
+
+Use the `globals` block to define [global variables](../variables/globals.md) and access them in the `global` namespace via `global.<variable-name>`. You can define globals at any level of your project hierarchy and set different globals for various configurations (a set of Terramate files specific to a directory). 
+
+When you define global variables across different configurations, Terramate applies a simple merge strategy: it merges `globals` with unique names and overrides `globals` with identical names using the more specific configuration.
```

Comment:
This could use further elaboration and some examples to make it clear.

For example, if I have two globals with the same name in the same stack, does it mean they will be merged, if yes, how, if now, will the conflict?

Reply:
Thanks for the input.
I've just included a line which mentions that you cannot redifine `globals` in the same configuration. Will that be good?